go/pkg/repository/disk.go

```diff
@@ -176,18 +186,18 @@ func (s *DiskRepository) ListRecursive(results chan<- ListResult, folder string)
 			close(results)
 			return
 		}
-		results <- ListResult{Error: err}
+		results <- ListResult{Error: errors.ReadError(err.Error())}
 	}
 	close(results)
 }
 
 func (s *DiskRepository) MatchFilenamesRecursive(results chan<- ListResult, folder string, filename string) {
-	err := filepath.Walk(path.Join(s.rootDir, folder), func(path string, info os.FileInfo, err error) error {
+	err := filepath.Walk(path.Join(s.rootDir, folder), func(p string, info os.FileInfo, err error) error {
```

Comment:
Yeah me too, it just seemed idiomatic in Go to avoid shadowing the `path` package. But I'm changing it to another [pattern](https://github.com/golang/tools/blob/master/godoc/vfs/mapfs/mapfs.go#L13) I've found, of aliasing the `path` package to `pathpkg`.

Reply:
Nice. I like.